4-Deployment/README.md

```diff
@@ -0,0 +1,341 @@
+2. Open the project in **VS Code**, select the **Azure** icon on the left hand side. Sign-in if you haven't already.
+
+![Icon](./ReadmeFiles/azure_icon.png)
+
+3. Follow the steps below to create a static web app. Note its URI once it's created.
+
+![Step1](./ReadmeFiles/step1.png)
+![Step2](./ReadmeFiles/step2.png)
+![Step3](./ReadmeFiles/step3.png)
+![Step4](./ReadmeFiles/step4.png)
+![Step5](./ReadmeFiles/step5.png)
+![Step6](./ReadmeFiles/step6.png)
+
+4. Once your static web app is created, Azure will add a [GitHub action](https://docs.github.com/en/actions/learn-github-actions) to your repository. After that, each new push to your repository will trigger a deployment cycle.
+
+![Step7](./ReadmeFiles/step7.png)
+![Step8](./ReadmeFiles/step8.png)
+
+#### Step 2: Update the app's registration
+
+1. Navigate back to the [Azure Portal](https://portal.azure.com).
+1. In the left-hand navigation pane, select the **Azure Active Directory** service, and then select **App registrations**.
+1. In the resulting screen, select the name of your application.
+1. From the *Branding* menu, update the **Home page URL**, to the address of your service, for example [https://reactspa1.z22.web.core.windows.net/](https://reactspa1.z22.web.core.windows.net/). Save the configuration.
+1. Add the same URI in the list of values of the *Authentication -> Redirect URIs* menu. If you have multiple redirect URIs, make sure that there a new entry using the App service's URI for each redirect URI.
+
+#### Step 3: Add environment variables to your static web app
+
+1. Navigate back to the [Azure Portal](https://portal.azure.com).
+1. Find the application you've created previously under **Azure Static Web Apps** and select it.
+1. Select the **Configuration** blade on the left hand side. Add the following **environment variables** there:
+    1. `CLIENT_ID`: enter the application ID (clientId) of `msal-react-spa` app copied from the Azure portal.
+    1. `TENANT_ID`: enter your Azure AD tenant ID.
+    1. `EXPECTED_SCOPES`: enter the name of the scope in APP ID URI that you've registered earlier.
+
+![Step9](./ReadmeFiles/step9.png)
+
+For more information, see [Configure application settings for Azure Static Web Apps Preview](https://docs.microsoft.com/azure/static-web-apps/application-settings#uploading-application-settings)
+
+## Explore the sample
+
+1. Open your browser and navigate to your deployed client app's URI, for instance: `https://reactspa1.z22.web.core.windows.net/`.
+1. Select the **Sign In** button on the top right corner. Choose either **Popup** or **Redirect** flow.
+1. Select the **Profile** button on the navigation bar. This will make a call to the Microsoft Graph API.
+1. Select the **FunctionAPI** button on the navigation bar. This will make a call to your web API.
+
+![Screenshot](./ReadmeFiles/screenshot.png)
```

Comment:
Should we update the screenshot?
This screenshot doesn't have the functions button, and it is pointed to localhost/hello instead of staticwebsite/api/hello

Reply:
updated